### ISSUE TITLE
Weights from MAISI in need of fine-tuning

### ISSUE THREAD

**Issue by Couch-bit** [AI-MED-AGH/robust-radiotherapy-planning#22]
It was decided that foundation models from [here](https://huggingface.co/nvidia/NV-Generate-CT) were going to be utilized. Since the data normalization strategy is being changed the VAE has to be retrained, the U-Net needs fine-tuning as the original network definition does not accept direct conditioning and is instead received from ControlNet in the form of residuals. The model also needs to adapt to the new VAE. This should be done:
- Train the VAE.
- After the VAE is set in stone, train the weights of U-Net with a warm-start from the foundation model weights.
- Document the training scripts along with necessary information.
